mobile-app/lib/ui/views/learn/widgets/quiz_widget.dart

```diff
@@ -92,16 +132,16 @@ class QuizWidget extends StatelessWidget {
     required int selectedAnswer,
     required bool? isCorrect,
     required ValueChanged<int> onChanged,
+    required int questionIndex,
   }) {
-    HTMLParser parser = HTMLParser(context: context);
-
     final isSelected = answerObj.key == selectedAnswer;
```

Comment:
Since we're passing `questionIndex` why not get the selected answer using that and `widget.questions`. This way we don't need to pass the variable